.travis.yml

```diff
@@ -1,57 +1,37 @@
 node_js:
-  - '0.10'
-  - '0.12'
+- '0.10'
+- '0.12'
 before_script:
-  - npm install grunt-cli -g
-  - export CI_BUILD_NUMBER=$(git rev-parse HEAD)
-
+- npm install grunt-cli -g
+- export CI_BUILD_NUMBER=$(git rev-parse HEAD)
 after_success:
-
-  # Temporary workaround for:
-  # https://github.com/travis-ci/travis-ci/issues/929
-
-  - python travis_after_all.py
-  - export $(cat .to_export_back) &> /dev/null
-  - |
-      # If all the tests pass in all the runtimes, make Travis
-      # automatically download and execute the following script
-
-      if [ "$BUILD_LEADER" == "YES" ]; then
-        if [ "$BUILD_AGGREGATE_STATUS" == "others_succeeded" ]; then \
-
-          # Clean up helper files
-          rm -rf travis_after_all.py .to_export_back && \
-
-          # The `update_site_branch.sh` script will run the build,
-          # and if that generates changes, it will commit them to the
-          # `master` branch:
-          #
-          #   * ensuring that the content from the `gh-pages/` directory
-          #     is always in sync with the rest of the content
-          #
-          #   * removing the need to execute the build step locally
-          #     everytime a change is made (especially in the case
-          #     of trivial changes such as typos)
-          #
-          # Note: The `update_site_branch.sh` script will only run
-          # if the commit was made to the `master` branch.
-
-          curl -sSL "https://raw.githubusercontent.com/h5bp-bot/scripts/0.6.1/update_site_branch.sh" |
-              bash -s -- --commands "grunt copy:gh-pages" \
-                         --commit-message "Hey server, this content is for you! [skip ci]" \
-                         --directory "gh-pages" \
-                         --distribution-branch "gh-pages" \
-                         --source-branch "master";
-        fi
-      fi
-
+- python travis_after_all.py
+- export $(cat .to_export_back) &> /dev/null
+- ! "# If all the tests pass in all the runtimes, make Travis\n# automatically download
+  and execute the following script\n\nif [ \"$BUILD_LEADER\" == \"YES\" ]; then\n
+  \ if [ \"$BUILD_AGGREGATE_STATUS\" == \"others_succeeded\" ]; then \\\n\n    # Clean
+  up helper files\n    rm -rf travis_after_all.py .to_export_back && \\\n\n    # The
+  `update_site_branch.sh` script will run the build,\n    # and if that generates
+  changes, it will commit them to the\n    # `master` branch:\n    #\n    #   * ensuring
+  that the content from the `gh-pages/` directory\n    #     is always in sync with
+  the rest of the content\n    #\n    #   * removing the need to execute the build
+  step locally\n    #     everytime a change is made (especially in the case\n    #
+  \    of trivial changes such as typos)\n    #\n    # Note: The `update_site_branch.sh`
+  script will only run\n    # if the commit was made to the `master` branch.\n\n    curl
+  -sSL \"https://raw.githubusercontent.com/h5bp-bot/scripts/0.6.1/update_site_branch.sh\"
```

Comment:
its about to be rewritten by @alrra to make it a lot simplier. I can make it local once that happens